RabbitMQ.Stream.Client/RawProducer.cs

```diff
@@ -122,12 +122,24 @@ private async Task Init()
                 {
                     foreach (var id in publishingIds.Span)
                     {
-                        _config.ConfirmHandler(new Confirmation
+                        try
                         {
-                            PublishingId = id,
-                            Code = ResponseCode.Ok,
-                            Stream = _config.Stream
-                        });
+                            _config.ConfirmHandler(new Confirmation
+                            {
+                                PublishingId = id,
+                                Code = ResponseCode.Ok,
+                                Stream = _config.Stream
+                            });
+                        }
+                        catch (Exception e)
```

Comment:
As written in the comment   The call is exposed to the user, so we need to catch any exception there could be an exception in the user code.  we need to trap `Exception` to avoid blocking the iteration.